SU2_CFD/include/numerics_simd/flow/diffusion/common.hpp

```diff
@@ -171,7 +171,8 @@ FORCEINLINE void addTauWall(Int iPoint, Int jPoint,
   Double tauWall_ij = (tauWall_i+tauWall_j) * isNormalEdge;
 
   /*--- Scale is 1 for those edges, i.e. tau is not changed. ---*/
-  Double scale = tauWall_ij / norm(tangentProjection(tau,unitNormal)) + (1.0-isNormalEdge);
+  Double scale =
+      tauWall_ij / fmax(norm(tangentProjection(tau,unitNormal)), EPS) + (1.0-isNormalEdge);
```

Comment:
:+1: 
These div-by-zero risks are actually a huge drawback of using y+ instead of y* as the non-dimensional length scale. I'll look into this 'soon'.